conf/battle/client.conf

```
@@ -173,3 +173,13 @@ macro_detection_punishment_time: 0
 // Macrochecker delay (per map)
 // Set to 0 to disable
 macrochecker_delay: 600000
+
+// Should companion pets/elementals/homunculi/mercenaries display the guild emblem of their master? (Note 3)
+// Uses unit types bit field values:
+// 0: None - Hide guild emblems for all companions
+// 4: Pets
+// 8: Homunculus
+// 16: Mercenary
+// 512: Elemental
+// Default: 0
+companion_show_guild_emblem: 0
```

```suggestion
companion_show_guild_emblem: 0

```
Missing empty newline at the end of the file.